modules/ocf/manifests/packages/firefox.pp

```diff
@@ -3,14 +3,20 @@
     stage => first,
   }
 
-  $firefox_pkg = $::lsbdistcodename ? {
-    'jessie' => 'firefox',
-    default  => 'firefox-esr',
+  if $::lsbdistcodename == 'jessie' {
```

Comment:
I'm trying with `apt-cache policy`. `apt-show-versions` is easier but installs a daily cronjob and an apt hook, so it seemed a bit heavy-weight for our purposes. But it's a great package to see where packages are installed from.